invenio_jsonschemas/jsonresolver.py

```diff
@@ -41,3 +43,8 @@ def jsonresolver_loader(url_map):
         "{0}/<path:path>".format(current_app.config['JSONSCHEMAS_ENDPOINT']),
         endpoint=current_app.extensions['invenio-jsonschemas'].get_schema,
         host=current_app.config['JSONSCHEMAS_HOST']))
+
+
+loader = json_loader_factory(LocalProxy(lambda: jsonresolver.JSONResolver(
```

Comment:
@jmartinm I'm afraid that this is not the right file to paste the loader implementation. You can move it to the state object:

```python
@cached_property
def loader(self):
    """TODO."""
    loader_ = import_string(self.app.config[...])  # or it might be already an instance (see https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/utils.py#L76-L87)
    if loader_ is None:
        loader_ = json_loader_factory(jsonresolver.JSONResolver(
            plugins=['invenio_jsonschemas.jsonresolver']
        ))
    return loader_
```